.gitignore

```diff
@@ -50,3 +50,6 @@ node_modules/
 *.log
 # Desktop Service Store
 *.DS_Store
+
+# virtual environments
+env/
```

Comment:
Could you add drop unrelate changes?

To ignore `env`,  you could add a `.gitignore` file inside `env`:
```
echo "*" > env/.gitignore
```

Or you could add `env/` to a global git ignore file.